app/views/layouts/_timeline.html.haml

```diff
@@ -2,130 +2,97 @@
 %script{:type => "text/javascript"}
   -# commented this to, for a workaround to close any open bubble from previous timeline when switching between timelines
   -# var tl;
-  - load_tl_now ||= false
-  ManageIQ.timelineFilter = #{!! @timeline_filter};
+  var element, timeline;
   function miqLoadTL() {
```

Comment:
Since you're rewriting this anyway, would you move the function definition to some js file outside this partial, and just call it from here with the needed data as params, please?